packages/example-app-base/src/components/ColorPalette/ColorPalette.tsx

```diff
@@ -94,27 +94,32 @@ class ColorPaletteBase extends React.Component<IColorPaletteProps, IColorPalette
           <div className={classNames.detailValues}>
             <span className={classNames.detailHex}>{hex}</span>
             {code && (
-              <>
-                <span className={classNames.detailCode} aria-describedby="code">
-                  {code.react.split('.')[1]}{' '}
-                  <TooltipHost
-                    tooltipProps={{
-                      onRenderContent: () => this._renderCodeDetails(code)
-                    }}
-                    closeDelay={500}
-                    id="code"
-                  >
-                    <Icon className={classNames.detailCodeInfoIcon} iconName="Info" />
-                  </TooltipHost>
-                </span>
-              </>
+              <span className={classNames.detailCode} aria-label={this._getCodeAriaLabel(code)}>
```

Comment:
Same issue here where the aria-describedby was invalid when the tooltip wasn't open. Adding an explicit aria-label fixes the issue.